gson/src/test/java/com/google/gson/internal/LinkedTreeMapSuiteTest.java

```diff
@@ -74,7 +74,8 @@ public static Test suite() {
             .named("nullValues=false")
             .createTestSuite();
 
-    TestSuite testSuite = new TestSuite("LinkedTreeMap");
+    // Use qualified class name to make it easier to find this test class in the IDE
+    TestSuite testSuite = new TestSuite(LinkedTreeMapSuiteTest.class.getName());
```

Comment:
For Eclipse this makes a difference: Otherwise it just shows the test suite name in the UI, and assertion stack traces are just Guava code, so there is no way to easily get to this test class.

Maven and IntelliJ automatically include the `LinkedTreeMapSuiteTest` class name.

The other test suite classes here are affected as well, but changing their name (e.g. "JsonObject#asMap") to the qualified class name would make the names derived for nested tests by Guava quite verbose. Could alternatively wrap the test suites in an additional `TestSuite` with the class as name (similar to here), but not sure if worth it.